app/views/manuals/_content_for_manual.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :extra_headers do %>
```

Comment:
That is a very good point! Since the content in that partial is to be included inside `<head>`, what do you think of renaming it to `_meta_content.html.erb`? Naming things is hard...